docs/DECLARING_DEPENDENCIES.md

```diff
@@ -185,6 +185,55 @@ https://docs.gradle.org/current/userguide/platforms.html#sub:bom_import).
 For Gradle 4 example, refer to [Gradle 4.6 Release Notes: BOM import](
 https://docs.gradle.org/4.6/release-notes.html#bom-import).
 
+## Guava Versions "-jre" or "-android"
```

Comment:
I need to think about this. I'm not sure this document is the right place for this.